//test exception 11;
int main()
{
	int* null = NULL;
	return *null;
}
